src/__tests__/to-be-invalid.js

```diff
@@ -1,3 +1,7 @@
+/**
+ * @jest-environment node
+ */
```

Comment:
Resolves this issue

I copy pasted from this PR

https://github.com/testing-library/jest-dom/pull/568/files?w=1#diff-aac3f7b77b88910a3cab3e23a102333b446c87359e633f41c0441c7738791d5b

```
  ● Test suite failed to run

    ReferenceError: TextEncoder is not defined

      1 |
      2 |
    > 3 | import {JSDOM} from 'jsdom'
        | ^
      4 | import {render} from './helpers/test-utils'
      5 |
      6 | /*
```